web-app/components/pages/login/login.jsx

```diff
@@ -82,6 +83,32 @@ export const Login = () => {
         },
         onSubmit: (values) => {
             console.log(JSON.stringify(values, null, 2));
+            axios({
+                method: 'post',
+                url: 'http://localhost:5000/api/authentication/login-attempt',
+                data: {
+                    email: values.email,
+                    password: values.password,
+                },
+            })
```

Comment:
And after destructuring the `values`, you can simply use `email` and `password` without have to be following with `values`
```suggestion
            axios({
                method: 'post',
                url: 'http://localhost:5000/api/authentication/login-attempt',
                data: {
                    email: email,
                    password: password,
                },
            })
```

And please note that in ES6, we can shorthand object key which its value is a variable that has the same name with the key by only write the key without value (please read about this feature in [this article](https://www.geeksforgeeks.org/shorthand-syntax-for-object-property-value-in-es6/)), so we can write it as
```suggestion
            axios({
                method: 'post',
                url: 'http://localhost:5000/api/authentication/login-attempt',
                data: {
                    email,
                    password,
                },
            })
```